### PR TITLE
Disable "Org Name" input when in single-tnt mode

### PR DESCRIPTION
**Pull request by ziyliu**
these occur when `SINGLE_TENANT_MODE: true`
login screen
Screenshot 2024-08-16 at 11 20 21 AM

forgot password 
Screenshot 2024-08-16 at 11 20 27 AM

reset password
Screenshot 2024-08-16 at 11 20 38 AM

### CODE REVIEW

File: public/settings/env-config.js
Comment (marcelosalloum) [inline public/settings/env-config.js:1]: Is this file something we should be committing? Cc @quietbits

File: webpack.common.js
Comment (marcelosalloum) [inline webpack.common.js:160]: I'm confused. Wasn't this supposed to be a boolean?

**Review by marcelosalloum — COMMENTED** [review]
I left a couple of questions, and I'd like to wait for Iveta's approval.

File: src/pages/ResetPassword.tsx
Comment (marcelosalloum) [inline src/pages/ResetPassword.tsx:127]: Rather than `disabled`, we should probably change its visibility to hidden instead.

File: public/settings/env-config.js
Comment (quietbits) [reply public/settings/env-config.js:1]: No, this file shouldn't be committed. It looks like `.gitignore` doesn't work for this file when it's modified.

File: src/pages/ResetPassword.tsx
Comment (quietbits) [reply src/pages/ResetPassword.tsx:127]: @marcelosalloum You meant that the field shouldn't be rendered, right? If we just change the visibility, there will be a gap in the markup. We need to render `null` instead.

```typescript
{ SINGLE_TENANT_MODE ? null : <Input ... /> }
```

File: webpack.common.js
Comment (ziyliu) [reply webpack.common.js:160]: values in `process.env` are always strings even if the env var has `true` or `false` set. Any value that's not `true` set to `process.env.REACT_APP_SINGLE_TENANT_MODE` will default to `false` when we generate the env configs in app!

**Review by quietbits — APPROVED** [review]

**Pull request merged**
